maven-build/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Maven build
+description: build a maven jar
+runs:
+  using: "composite"
+  steps:
+    - id: get-build-parameters
+      run: |
+        branch_name=$(echo ${{ github.ref }} | cut -d "/" -f 3)
+        revision=$branch_name-$(git log -n1 --date=format-local:%G%m%d-%H%M --format='%ad' | tr -d " ")
+        sha=$(git log -n1 --format='%h'| tr -d " ")
+        build_number=${{ github.run_number }}
+        build_tag=build.$branch_name-$build_number
+        echo "BRANCH_NAME=$branch_name" >> $GITHUB_ENV
+        echo "REVISION=$revision" >> $GITHUB_ENV
+        echo "SHA=$sha" >> $GITHUB_ENV
```

Comment:
There is `GITHUB_SHA` ... that said this is set to different values depending on the event type  
https://docs.github.com/en/actions/reference/events-that-trigger-workflows

![image](https://user-images.githubusercontent.com/1393946/113294677-99cf2900-92f7-11eb-8cee-db19668cbb6f.png)

Reply:
I will give you the same answer than here https://github.com/BrandwatchLtd/slipstream-actions/pull/31/files/7f0a1ad82b19d6c37c250840a0a663f50954776e#r605639160

Maybe let's leave it as close as the jenkins globale pipeline file for now and the backenders or someone who knows better how this jar should be tagged update it in the future